### PR TITLE
Make fetch-crl success optional, but give a loud warning on failure

### PR DESCRIPTION
**Pull request by matyasselmeci**

### CODE REVIEW

File: osg_configure/configure_modules/misc.py [hunk 1]
```diff
@@ -108,8 +108,10 @@ def configure(self, attributes):
 
         # run fetch-crl script
         if not utilities.fetch_crl():
-            self.log("Error while running fetch-crl script", level=logging.ERROR)
-            raise exceptions.ConfigureError('fetch-crl returned non-zero exit code')
+            self.log("Error while running fetch-crl script", level=logging.WARNING)
+
+        if not utilities.crls_exist():
+            self.log("No CRLs exist and Fetch-CRL couldn't download any.  GSI authentication is likely to fail.  To attempt another download, run 'fetch-crl'.", level=logging.WARNING)
```
Comment (brianhlin) [inline osg_configure/configure_modules/misc.py:114]: Is this message true? It's possible that there was an error in fetching just one CRL, which could potentially be fine

Comment (matyasselmeci) [reply osg_configure/configure_modules/misc.py:114]: `if not utilities.crls_exist()...`
That message is only printed if _no_ CRLs exist, i.e. there are no *.r0 files.

**Comment by matyasselmeci** [comment]
Created SOFTWARE-4364 (https://opensciencegrid.atlassian.net/browse/SOFTWARE-4364)

**Review by brianhlin — APPROVED** [review]
LGTM but could you disable Travis by removing `travis.yml` in a separate PR before merging this one?

**Pull request merged**